src/test/java/Homework18.java

```diff
@@ -0,0 +1,44 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+        public class Homework18 extends BaseTest {
+        @Test
+        public void playSong() throws InterruptedException {
+
+            navigateToPage();
+            provideEmail("john.wolschleger@testpro.io");
+            providePassword("te$t$tudentb49");
+            clickSubmit();
+            Thread.sleep(2000);
+            clickPlayNextSongBtn();
+            clickPlaySongBtn();
+
+}
+        public void clickPlayNextSongBtn() throws InterruptedException {
+        WebElement playNextSong = driver.findElement(By.xpath("//*[@id='mainFooter']/div[1]/i[2]"));
+            playNextSong.click();
+            Thread.sleep(2000);
+        }
+        public void clickPlaySongBtn() throws InterruptedException {
+            WebElement playSong = driver.findElement(By.xpath("//*[@id='mainFooter']/div[1]/span/span[2]"));
+            playSong.click();
+            Thread.sleep(2000);
+        }
+               public void expectedVisualizerBanner() throws InterruptedException {
+            WebElement visualizerBanner = driver.findElement(By.xpath("//*[@id=\"vizContainer\"] || //*[@id=\"mainFooter\"]/div[1]/span/span[2]/i"));
+
+            Assert.assertTrue(visualizerBanner.isDisplayed());
+          }
+        }
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._